src/material-elements/material-textarea/material-textarea.tag

```diff
@@ -22,6 +22,8 @@
             if(opts.maxRows) this.refs.mirror.style.maxHeight = opts.maxRows*this.refs[this.n].getBoundingClientRect().height + 'px';
             this.n = opts.ref||'default-textarea';
             // Defaults
+            this.value = opts.value || '';
+            this.mirror.innerHTML = this.format(this.value);
```

Comment:
Be carefull, this is not compatible with riot v3 and the latest version of riot-mui.

Please see [riot v3 breaking changes](http://riotjs.com/guide/migration-from-riot2/#breaking-changes)